build.cake

```diff
@@ -165,7 +165,25 @@ MSBuildSettings CreateSettings()
     settings.WithProperty("DebugType", "pdbonly");
 
     if (IsRunningOnWindows())
-        settings.ToolVersion = MSBuildToolVersion.VS2017;
+    {
+        // Find MSBuild for Visual Studio 2017
+        DirectoryPath vsLatest  = VSWhereLatest();
+        FilePath msBuildPathX64 = (vsLatest==null) ? null
+                                    : vsLatest.CombineWithFilePath("./MSBuild/15.0/Bin/MSBuild.exe");
+
+        // Find MSBuild for Visual Studio 2019
+        if ( msBuildPathX64 != null && !FileExists(msBuildPathX64))
+            msBuildPathX64 = vsLatest.CombineWithFilePath("./MSBuild/Current/Bin/MSBuild.exe");
```

Comment:
The x64 path is `\MSBuild\Current\Bin\amd64\MSBuild.exe`. The only time I've seen the bitness of the MSBuild process matter is when building VSIXes.

Reply:
I just copied and pasted from the adapter project. This fix is a mix of fixes from there and from the ones you did for the console project. I will make the suggested changes.